ecosystem/gstreamer_plugin/gst_mtl_st30p_rx.c

```diff
@@ -146,7 +146,7 @@ static void gst_mtl_st30p_rx_class_init(Gst_Mtl_St30p_RxClass* klass) {
   gstbasesrc_class = GST_BASE_SRC_CLASS(klass);
 
   gst_element_class_set_metadata(
-      gstelement_class, "MtlRxSt20Src", "Src/Audio",
+      gstelement_class, "MtlRxSt30Src", "Src/Audio",
       "MTL transmission plugin for SMPTE ST 2110-20 standard (uncompressed video)",
```

Comment:
-> - 30 ?

Reply:
Yes an overstie there were others but all should be fixed now thanks